pingouin/multivariate.py

```diff
@@ -254,3 +254,102 @@ def multivariate_ttest(X, Y=None, paired=False):
     stats = {'T2': t2, 'F': fval, 'df1': df1, 'df2': df2, 'pval': pval}
     stats = pd.DataFrame(stats, index=['hotelling'])
     return _postprocess_dataframe(stats)
+
+def box_m (data, vars=None, group=None, alpha=.001):
```

Comment:
Is there a reason why the significance level is set to 0.001? Is that standard for the Box M test?

Reply:
0.001 is commonly used. I cite:
> In addition, Box’s M has very little power (Cohen, 2008) for small sample sizes; if your small-sample result is not significant, it doesn’t necessarily indicate that the covariance matrices are equal. The test has also been criticized for being overly sensitive for large sample sizes. In other words, it will report a statistically significant result when one doesn’t actually exist. To address this particular issue, a smaller alpha level (e.g. .001) is recommended (Hahs-Vaughn, 2016).

The reference is **Hahs-Vaughn, D. (2016). Applied Multivariate Statistical Concepts. Taylor & Francis.**